apps/readest-app/.claude/memory/file-sync-converge-5900.md

```
@@ -0,0 +1,115 @@
+---
+name: file-sync-converge-5900
+description: "#5900 WebDAV/file sync never converged across devices; root causes and the fix, MERGED as PR #5905"
+metadata: 
+  node_type: memory
+  type: project
+  originSessionId: 1d46dba2-18c6-424a-b6f7-8e19dc3004f5
+  modified: 2026-08-27T14:19:36.946Z
+---
+
+#5900 "WebDAV Full Sync does not converge library state across devices" (3 devices:
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)